libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/dh-processes-table/dh-processes-table.component.scss

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+:host {
+  display: block;
+  padding: var(--watt-space-l) 0;
+}
+
+.mat-column-expand,
+.mat-column-hasErrors {
+  max-width: 52px;
+  padding: 10px;
+}
+
+.mat-column-expand watt-icon {
+  cursor: not-allowed; // TODO: Change this when the expand functionality has been implemented
+}
+
+// Add margin left for any span following a watt-icon
+watt-icon + span {
+  margin-left: var(--watt-space-s);
+}
```

Comment:
question: Why is this necessary? Do you have a screenshot of what is the effect without it?

Reply:
Wouldn't it work the same if we added `margin-right` to the `watt-icon` component, so the `span` is skipped?

Reply:
It would, but I have a watt-icon for the "hasDetailsErrors" column aswell and that shouldn't have any margins. Figured this was the easiest way to apply it for all icons that have a span next to them.

Reply:
In this case we can make the intention clearer by using the `.date-icon` class and adding `margin-right` to it.